Procfile

```diff
@@ -1,4 +1,5 @@
-web:              node bin/server.js            production
-reaper:           node bin/reaper.js            production
-expireArtifacts:  node bin/expire-artifacts.js  production
-retireTasks:      node bin/retire-tasks.js      production
\ No newline at end of file
+web:              npm run start
+claimReaper:      npm run claim-reaper
+deadlineReaper:   npm run deadline-reaper
+expireArtifacts:  npm run expire-artifacts
+expireTask:       npm run expire-tasks
```

Comment:
yeah, it's easiest to scale that way...

Note, we might make expireArtifacts a part of expireTask in the future...